src/components/TF2/Player/playerutils.tsx

```diff
@@ -78,14 +85,39 @@ function displayColor(
 
   if (you) return hexToRGB(playerColors['You'], ALPHA);
 
-  if (!verdict || verdict.includes('None') || verdict.includes('Player'))
-    return hexToRGB(playerColors['Player'], ALPHA);
+  if (convicted) return hexToRGB(playerColors['Convict'], ALPHA);
+
+  if (!verdict || verdict.includes('None') || verdict.includes('Player')) {
+    if (player.tags?.includes('Friend'))
+      return hexToRGB(playerColors['Friend'], ALPHA);
 
-  if (convicted) return hexToRGB(playerColors['Cheater'], ALPHA);
+    if (hasCheaterFriendsInLobby(player, cheatersInLobby))
+      return hexToRGB(playerColors['FriendOfCheater'], ALPHA);
+
+    return hexToRGB(playerColors['Player'], ALPHA);
+  }
 
   return hexToRGB(playerColors[verdict], ALPHA);
 }
 
+function getCheatersInLobby(

```

Comment:
Minor style complaint:
Could probably have a more descriptive name like `getCheaterFriendsInLobby`, that would be more consistent with `hasCheaterFriendsInLobby` and makes it clear that it's not *all* the cheaters in lobby, but just those who are friends of a player.

Reply:
Fixed